Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -496,6 +536,69 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .AddParameter("string", _("Variable"))
       .SetHidden();
 
+  obj.AddAction("ObjectVariablePush",
+                _("Push variable on an object array"),
+                _("Appends a variable at the end of a object array variable."),
```

Comment:
I think (here and below), that the correct syntax is "appends [something] to the end of".

Also everywhere: a object => an object